recipes/libiconv/all/conanfile.py

```diff
@@ -5,7 +5,7 @@
 import os
 import functools
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.47.0"
```

Comment:
```suggestion
required_conan_version = ">=1.45.0"
```
is_msvc was actually added in [1.45.0](https://docs.conan.io/en/latest/changelog.html#feb-2022)